crates/milli/src/update/new/indexer/mod.rs

```diff
@@ -93,6 +93,13 @@ where
         },
     );
 
+    tracing::debug!(
+        "Indexation allocated memory metrics - \
+        Total BBQueue size: {total_bbbuffer_capacity}, \
+        Total extractor memory: {:?}",
+        grenad_parameters.max_memory,
+    );
```

Comment:
These won't change over the course of Meilisearch running, do you think it would be possible to only log it once?